src/components/brand/BrandIllustration.js

```diff
@@ -0,0 +1,164 @@
+import React, { Fragment } from 'react';
+import mainScreen from "../../assets/images/forYouScreen.png";
```

Comment:
almost a direct copy of the HeroIllistation.js except this line, it points to the other .png image. If the future this could be refactored to have the image file as a parameter and then we would only need Illistration.js file.